### PR TITLE
Added Location (city and state) and updated Accomplishments form

### PR DESCRIPTION
**Pull request by BayLadyCoder**
All tests passed, but there is a warning in the console when open a form to update (in experience).

_Warning: Failed prop type: Invalid prop `experience.start_year` of type `number` supplied to `AddOrEditExperienceForm`, expected `string`._

Tried to find the differences, but the code that could be the problem look the same to me compared to 'dev' but in 'dev' it doesn't give any error (I tested and compared to [http://dev-app.baltimorecorps.org/](url)).

### CODE REVIEW

File: src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js [hunk 1]
```diff
@@ -133,6 +139,11 @@ const AddOrEditExperienceForm = ({ experience, onSubmit, handleCancel, classes }
   if (typeof values.start_year === String) {
     values.start_year = null;
   }
+
+  if (experience.type === 'Accomplishment') {
+    values.end_month = 'none';
+    values.end_year = 0;
+  }
```
Comment (dfkoh) [inline src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js]: This feels like it should happen in handleFormSubmit, and also like it should happen in general whenever values.end_month and values.end_year are null? It seems like in general we need a cleaner way of handling end_month and end_year in this module

Comment (dfkoh) [reply src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js]: It occurs to me that fixing up how end_month and end_year work isn't super related to the PR, though, so it probably shouldn't block the merge of this specific thing

File: src/modules/Experiences/ExperiencesList/ExperiencesListItem.js [hunk 2]
```diff
@@ -13,12 +13,12 @@ import { formatMonthYearDate, getWorkLength } from './helpers';
...
   const title =
-    `${experience.host}, ` +
-    (experience.type === 'Education'
+    experience.type === 'Education'
       ? `${experience.degree} in ${experience.title}`
-      : `${experience.title}`);
+      : `${experience.title}`;
   const showEndDate = experience.type !== 'Accomplishment';
   const showAchievements = experience.type !== 'Accomplishment';
+  const showWorkLength = experience.type !== 'Accomplishment';
```
Comment (dfkoh) [inline src/modules/Experiences/ExperiencesList/ExperiencesListItem.js]: It's a little awkward that these settings are getting defined in two different places: here and in `configureForm` in the form code. Maybe we could combine those into a helper function somehow?

Comment (BayLadyCoder) [reply src/modules/Experiences/ExperiencesList/ExperiencesListItem.js]: done

File: src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js (hunk 1 above)
Comment (BayLadyCoder) [reply src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js]: Created isEndDateNull() in helper.js to handle any (end_month/year === null) situation

**Pull request merged**